app/_components/inputs/Commodities.tsx

```diff
@@ -33,31 +40,30 @@ const CommoditiesField: React.FC<CommodityProps> = ({
     }));
   }
 
+  const fieldOptions: FieldOptions = {};
+  if (isMulti) {
+    fieldOptions.isMulti = true;
+  }
+
```

Comment:
Not a blocker: What other options do you anticipate for this? If not, you can just pass it in the Select component.

Reply:
@nitishdhar You can't pass `isMulti={false}` on a react-select, it either accepts `true` or simply can't be there. So the simplest way of dealing with this, I figured, would be to populate an options object and spread it onto the Select, so if isMulti is set to false via the component prop it just wouldn't be included.

![image](https://github.com/ed-pilots-network/frontend/assets/319883/7880293b-147f-4a30-b011-a4ea10f04eb1)

Reply:
Nah, this is good then. Thanks for clarifying. Weird they don't take false there.